assets/src/js/commands/admin-commands.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Admin Commands
+ *
+ * Core WordPress commands for Secure Custom Fields administration.
+ * This file registers navigation commands for all primary SCF admin screens,
+ * enabling quick access through the WordPress commands interface (Cmd+K / Ctrl+K).
+ *
+ * @since 6.5.0
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createElement } from '@wordpress/element';
+import { Icon } from '@wordpress/components';
+import { dispatch } from '@wordpress/data';
+import { createQueue } from '@wordpress/priority-queue';
+
+/**
+ * Initialize deferred execution queue and context
+ */
+const queue = createQueue();
+const context = {};
+
+/**
+ * Register admin commands for SCF
+ */
+queue.add( context, () => {
+	if ( ! dispatch( 'core/commands' ) || ! window.acf?.data ) {
+		return;
+	}
+
+	const commandStore = dispatch( 'core/commands' );
+	const adminUrl = window.acf?.data?.admin_url || '';
```

Comment:
which script defines `window.acf` ?

Reply:
The object is defined here:
https://github.com/WordPress/secure-custom-fields/blob/8b71adbd32aa0970c2c693a7c5e150ae2942289b/assets/src/js/_acf.js#L15

And the script is loaded here:
https://github.com/WordPress/secure-custom-fields/blob/8b71adbd32aa0970c2c693a7c5e150ae2942289b/includes/assets.php#L152

https://github.com/WordPress/secure-custom-fields/blob/8b71adbd32aa0970c2c693a7c5e150ae2942289b/includes/assets.php#L237